nixos/modules/services/monitoring/prometheus/exporters/rtl_433.nix

```diff
@@ -0,0 +1,75 @@
+{ config, lib, pkgs, options }:
+
+let
+  cfg = config.services.prometheus.exporters.rtl_433;
+in
+{
+  port = 9550;
+
+  extraOpts = let
+    mkMatcherOptionType = field: description: with lib.types;
+      listOf (submodule {
+        options = {
+          name = lib.mkOption {
+            type = str;
+            description = "Name to match.";
+          };
+          "${field}" = lib.mkOption {
+            type = int;
+            inherit description;
+          };
+          location = lib.mkOption {
+            type = str;
+            description = "Location to match.";
+          };
+        };
+      });
+  in
+  {
+    rtl433Flags = lib.mkOption {
+      type = lib.types.str;
+      default = "";
+      example = "-R 19";
+      description = ''
+        Flags passed verbatim to rtl_433_prometheus binary.
```

Comment:
That seems incorrect -- the flags are passed to rtl_433, not rtl_433_prometheus.

Reply:
Good catch, fixed.